docs/README-AWS.md

```diff
@@ -398,70 +479,99 @@ inputs:
+For these use cases, setting the `es_datastream_name` field in the configuration file is optional.
 
-If the `es_datastream_name` is not specified and it cannot be matched with any of the above AWS services then the dataset will be set to "generic" and the namespace to "default" pointing to the data stream name "logs-generic-default".
+For most of the other use cases, you will need to set the `es_datastream_name` field in the configuration file to route the data to a specific data stream or index. This value should be set in the following use cases:
 
-## Setting up S3 event notification to SQS
-In order to set up an S3 event notification to SQS please look at the official documentation: https://docs.aws.amazon.com/AmazonS3/latest/userguide/NotificationHowTo.html
+- You want to write the data to a specific index, alias, or custom data stream, and not to the default integration data streams. This can help some users to use existing Elasticsearch assets like index templates, ingest pipelines, or dashboards, that are already set up and connected to business processes.
+- When using `Kinesis Data Stream`, `CloudWatch Logs subscription filter` or `Direct SQS message payload` inputs. Only the `S3 SQS Event Notifications input` method supports automatic routing to default integration data streams for several AWS service logs.
+- When using `S3 SQS Event Notifications input` but where the log type is something *other than* AWS CloudTrail (`aws.cloudtrail`), Amazon CloudWatch Logs (`aws.cloudwatch_logs`), Elastic Load Balancing (`aws.elb_logs`), AWS Network Firewall (`aws.firewall_logs`), Amazon VPC Flow (`aws.vpcflow`), and AWS Web Application Firewall (`aws.waf`).
 
-The event type to set up in the notification should be `s3:ObjectCreated:*`
+If the `es_datastream_name` is not specified, and the log cannot be matched with any of the above AWS services, then the dataset will be set to `generic` and the namespace set to `default`, pointing to the data stream name `logs-generic-default`.
 
-The Elastic Serverless Forwarder Lambda needs to be provided extra IAM policies in order to access S3 and SQS resources in your account: please refer to [Lambda IAM permissions and policies](#lambda-iam-permissions-and-policies).
+## Setting up S3 event notifications for SQS
+To set up an S3 event notification for SQS, see the [Amazon S3 docs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/NotificationHowTo.html).
 
-## Error handling
-There are two kind of errors that can happen during the execution of the Lambda:
-1. Errors before the ingestion phase started
+You need to set the notifications up for the `s3:ObjectCreated:*` event type.
+
+The Elastic Serverless Forwarder Lambda function needs to be provided extra IAM policies in order to access S3 and SQS resources in your account. For more info, see [Lambda IAM permissions and policies](#lambda-iam-permissions-and-policies).
```

Comment:
1. the internal link is now `[Permissions and policies](#permissions-and-policies)`
2. indeed there is no need to provide extra IAM policies to the Lambda anymore

```
The extra IAM policies required in order to access S3 and SQS resources in your account by the Elastic Serverless Forwarder Lambda are set up automatically. For more info, see [Permissions and policies](#permissions-and-policies).
```